backend/open_api_spec.yml

```diff
@@ -0,0 +1,790 @@
+      security:
+        - cxguserCookie: []
+      responses:
+        "201":
+          description: Created
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  key:
+                    type: string
+                    required: true
+                  id:
+                    type: string
+                    required: true
+        "401":
+          $ref: "#/components/responses/401"
+    get:
+      summary: Fetch the identifier of the curator's active API keys
+      security:
+        - cxguserCookie: []
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: string
+        "401":
+          $ref: "#/components/responses/401"
+    delete:
+      summary: Delete the curator's API key
+      security:
+        - cxguserCookie: []
+      responses:
+        "202":
+          $ref: "#/components/responses/202"
+        "401":
+          $ref: "#/components/responses/401"
+
+  /curation/auth/token:
+    post:
+      summary: Generate an access token
+      security:
+        - apiKey: []
+      responses:
+        "201":
+          description: Created
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  access_token:
+                    type: string
+        "401":
+          $ref: "#/components/responses/401"
+
+  /curation/collections:
+    get:
+      description: Fetch all Collections metadata. Authentication required for private collections.
+      security:
+        - cxguserCookieLenient: []
+      parameters:
+        - $ref: "#/components/parameters/query_visibility"
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  collections:
+                    type: array
+                      items:
+                        type: object
+                          required:
+                            - id
+                            - name
+                            - visibility
+                            - contact_name
+                            - contact_email
+                            - curator_name
+                            - revised_at
+                            - created_at
+                            - links
+                            - datasets
+                          properties:
+                            id:
+                              $ref: "#/components/schemas/collection_uuid"
+                            name:
+                              type: string
+                            visibility:
+                              $ref: "#/components/schemas/visibility"
+                            contact_name:
```

Comment:
General question: For names of humans (e.g. `contact_name`, `curator_name`) do we want/expect `<first name> <last name>` or `<last name>, <first name>`? Should this be documented in a description somewhere?

Reply:
We're _preferring_ `<first name> <last name>` for the contact, but obviously cannot enforce it. I've seen a recent collection where the contact is an info mailing list for Kidney Precision Medicine Project (KPMP) - see https://cellxgene.cziscience.com/collections/bcb61471-2a44-4d00-a0af-ff085512674c.

The `curator_name` is _automagically_ set by the BE using Auth0.